### ISSUE TITLE
Unable to obtain accessToken via ebx-linkedin-sdk 

### ISSUE THREAD

**Issue by shrinidhid** [ebx/ebx-linkedin-sdk#123]
Code used:

```
DefaultLinkedInClient client = new DefaultLinkedInClient(Version.DEFAULT_VERSION);
        LinkedInClient.AccessToken accessToken = client.obtainUserAccessToken("client_id", "client_secret",
                "redirect url", code);
```

The below is the error at Line 343(String response) DefaultLinkedInClient.class

ERROR:

```
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN""http://www.w3.org/TR/html4/strict.dtd">
<HTML><HEAD><TITLE>Bad Request</TITLE>
<META HTTP-EQUIV="Content-Type" Content="text/html; charset=us-ascii"></HEAD>
<BODY><h2>Bad Request - Invalid Header</h2>
<hr><p>HTTP Error 400. The request has an invalid header name.</p>
</BODY></HTML>
```

Version:
		<dependency>
		  <groupId>com.echobox</groupId>
		  <artifactId>ebx-linkedin-sdk</artifactId>
		  <version>2.0.0</version>
		</dependency>

**Comment by diepjy** [comment]
Hi shrinidhid,

Thanks for raising the issue, this issue has been marked as a bug. An additional header `Content-Type: application/x-www-form-urlencoded` is required to make this request as outlined in the [documentation](https://docs.microsoft.com/en-us/linkedin/shared/authentication/authorization-code-flow?context=linkedin/context).

If you would to raise a PR to add this API you are more than welcome to by following the [contributions guide](https://github.com/ebx/ebx-linkedin-sdk/blob/master/CONTRIBUTING.md).

**Comment by mcadecio** [comment]
Hi I would like to fix this issue. Could you kindly provide access for me to raise an PR?

**Comment by mcadecio** [comment]
Hi @diepjy ,

Please provide some feedback on the PR when you are available.

Thanks in advance.

**Comment by joergrech** [comment]
Hi, I have the same problem - is it still unfixed or not merged with the production branch? I can access the linkedin page via curl - so it should work.

**Comment by mcadecio** [comment]
Hi @jack7ellis / @kennethwong-hc / @diepjy , I have just update the PR so it is up to date with dev branch. Can I get a review?

**Comment by mcadecio** [comment]
> Hi, I have the same problem - is it still unfixed or not merged with the production branch? I can access the linkedin page via curl - so it should work.

@joergrech Still waiting on a review.

**Comment by mcadecio** [comment]
Hey @joergrech / @shrinidhid  this is fixed in version 3.1.3. Thanks @kennethwong-hc .

**Comment by kennethwong-hc** [comment]
Resolved in 3.1.3
